files/nbde_client/nbde_client-hook.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# disable_inird_connections() will disable autoconnect for the active
+# connections within the initramfs, so that these will not clash with
+# the system network connections once the flush is performed and
+# the network configuration is applied to the system.
+disable_inird_connections() {
+  nmcli -t -f NAME connection show --active | while read -r _c; do
```

Comment:
@liangwen12year is this the recommended way to disable autoconnect?

Reply:
Hm it looks like this only works with newer dracut, where NM runs as a daemon wirh dbus: https://github.com/dracutdevs/dracut/commit/112f03f9e225a790cbc6378c70773c6af5e7ee34 -- this is causing issues with e.g. CentOS Stream 8